tools/src/h5dump/h5dump.c

```diff
@@ -1368,10 +1375,15 @@ main(int argc, char *argv[])
     /* Initialize indexing options */
     h5trav_set_index(sort_by, sort_order);
 
-    if (use_custom_vol_g || use_custom_vfd_g) {
-        if ((fapl_id = h5tools_get_fapl(H5P_DEFAULT, use_custom_vol_g ? &vol_info_g : NULL,
-                                        use_custom_vfd_g ? &vfd_info_g : NULL)) < 0) {
-            error_msg("unable to create FAPL for file access\n");
+    if ((fapl_id = h5tools_get_fapl(H5P_DEFAULT, use_custom_vol_g ? &vol_info_g : NULL,
```

Comment:
Same comment here about this now always creating a fapl, which affects the logic used for `h5tools_fopen`.